frappe/www/login.py

```diff
@@ -15,8 +15,12 @@
 no_cache = True
 
 def get_context(context):
+	redirect_to = frappe.local.request.args.get("redirect-to")
+
 	if frappe.session.user != "Guest":
-		frappe.local.flags.redirect_location = "/" if frappe.session.data.user_type=="Website User" else "/desk"
+		if not redirect_to:
+			redirect_to = "/" if frappe.session.data.user_type=="Website User" else "/desk"
```

Comment:
Should `/` be `/me` for Website User? because / will redirect to home page or login if home page is not set. If / redirect to /login it causes an infinite loop resulting to "Too many redirects"